frontend/src/axios.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export const axiosInstance = axios.create({
```

Comment:
You can still use the name `axios` instead of `axiosInstance`. It's more understandable.